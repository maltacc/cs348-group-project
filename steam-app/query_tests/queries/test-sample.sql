SELECT name, genres, price
FROM games
WHERE name like '%fake game 3%'
  AND price < 20.00;