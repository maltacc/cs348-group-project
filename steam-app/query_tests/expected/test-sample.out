name	genres	price
fake game 3	RPG	0.00
fake game 3	RPG	0.00
